_posts/2022-11-21-changelog-183.markdown

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #183"
+date: 2022-11-21 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#9694] Enable [derive](https://doc.rust-lang.org/reference/procedural-macros.html#derive-macros) procedural macro expansion in [nightly plugin builds](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-quick-start.html#install-nightly).
+          To turn it off, disable `org.rust.macros.proc.derive` [experimental feature](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-faq.html#experimental-features)
+
+* [#9641], [#9699] Support [Intention and quick-fixes preview](https://blog.jetbrains.com/idea/2022/09/intellij-idea-2022-3-eap/#Intention_action_preview_enabled_by_default) in 2022.3
+
+    TODO: gif
+
+
+* [#7353] Support [intra-doc links](https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html). Please note that currently we support links in format `[text](link)`, but not `[link]`
+
+    TODO: gif
+
+* [#9649] Enable native Rust support in [MSVC LLDB](https://blog.jetbrains.com/clion/2019/06/clion-2019-2-eap-msvc-debugger-unused-includes-check-and-more/#msvc_debug) in 2022.3 IDE builds on Windows. Currently, the most visible improvement of the native support is the proper displaying of Rust primitive type names, such as `u64` instead of `unsigned long long`
+
+    | Before | After |
+    |-|-|
+    | ![image](/assets/posts/changelog-183/msvc-lldb-before.png) | ![image](/assets/posts/changelog-183/msvc-lldb-after.png) |
+
+* [#8958] Provide support for proper build script evaluation and procedural macro expansion on Windows and Linux with arm64 CPU
+
+* [#9350] Support renaming derive procedural macros
```

Comment:
We usually add link to refactoring action, its shortcut and link to documents when mention any refactoring